### PR TITLE
Perk card for hackathon grants

### PR DESCRIPTION
**Pull request by cjdenio**
This is currently locked behind a feature flag for Assemble stuff.

Screen Shot 2022-08-03 at 3 44 53 PM

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -307,4 +307,16 @@ def development_mode_flavor
...
+    query = {}
+    params.each do |key, value|
+      query["prefill_#{key}"] = value
+    end
+    hide.each do |field|
+      query["hide_#{field}"] = "true"
+    end
+
+    "https://airtable.com/#{id}?#{URI.encode_www_form(query)}"
+  end
```
Comment (exu3) [inline app/helpers/application_helper.rb]: This is great, thanks for adding this.

File: app/views/events/promotions.html.erb [hunk 2]
```diff
@@ -6,6 +6,46 @@
   Promotions &amp; perks
 </h1>
 
+<% if @event.hackathon? && Flipper.enabled?(:assemble_woot, @event) %>
```
Comment (exu3) [inline app/views/events/promotions.html.erb]: Should we also add a flag for US/CA events?

File: app/views/events/promotions.html.erb [hunk 3]
```diff
@@ -6,6 +6,46 @@
...
+      <h3 class="h1 p2 my0 flex items-center smoke">💵 Hackathon Grant <span class="badge bg-success">New!</span></h3>
+      <%= link_to 'Apply',
+        organizer_signed_in? ? airtable_form("shr15T0RefCkUSace", {"Event Name" => @event.name, "Bank URL" => event_url(@event), "Email" => current_user.email}, ["Event Name", "Bank URL"]) : root_path,
+        class: "btn mr1 #{'disabled' unless organizer_signed_in?}",
+        target: :_blank
+      %>
+    </div>
+
+    <p>
+      Hack Club is partnering with <a href="https://firstinspires.org" target="_blank"><i>FIRST</i></a> to offer <strong>$250-$500</strong> grants to in-person high-school hackathons happening this semester.
```
Comment (exu3) [inline app/views/events/promotions.html.erb]:
```suggestion
      Hack Club Bank is partnering with <a href="https://firstinspires.org" target="_blank"><i>FIRST</i></a> to offer <strong>$250-$500</strong> grants to in person high school hackathons happening this semester (through December 31, 2022).
```

File: app/views/events/promotions.html.erb [hunk 4]
```diff
@@ -6,6 +6,46 @@
...
+    </p>
+
+    <div class="flex">
+      <div class="column">
+        <h3 class="mb0 mt0">Requirements</h3>
+        <ul class="pl2">
+          <li>Proof of venue, such as a scan of a contract.</li>
+          <li>Event must be fully in-person— not virtual or hybrid.</li>
```
Comment (exu3) [inline app/views/events/promotions.html.erb]:
```suggestion
          <li>Event must be fully in person—not virtual or hybrid.</li>
```

File: app/views/events/promotions.html.erb [hunk 5]
```diff
@@ -6,6 +6,46 @@
...
+          <li>Hand-made event website*</li>
```
Comment (exu3) [inline app/views/events/promotions.html.erb]:
```suggestion
          <li>Handmade event website*</li>
```

**Review by exu3 — APPROVED** [review]
The card looks great! Left some comments.


There seems to be a little gap between the border and the banner image.
Screen Shot 2022-08-02 at 10 44 29 AM

File: app/views/events/promotions.html.erb (hunk 2 above)
Comment (cjdenio) [reply app/views/events/promotions.html.erb]: Done!

**Review by maxwofford — APPROVED** [review]
This looks great– love the colored icons.

**Review by exu3 — APPROVED** [review]
Small thing: is there a way we could communicate that these are just examples (and not strict guidelines) of what a handmade site is or isn't? Eg. there are lots of other website builders (Weebly, Webflow, Google sites) that people might think are ok because they're not listed with the ⛔ icon here.

Screen Shot 2022-08-05 at 4 09 56 AM

Approving because the above comment is just a nice-to-have; the card looks great!

**Comment by cjdenio** [comment]
Merging with verbal approval from @maxwofford.

**Pull request merged**